bin/ddo

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-bin/dc run --rm --entrypoint /entrypoint.sh -e IS_DDO=1 django_shell $*
+bin/dc run --rm -e LOCAL_USER_ID=`id -u $USER` -e LOCAL_GROUP_ID=`id -g $USER` --entrypoint /entrypoint.sh -e IS_DDO=1 django_shell $*
```

Comment:
this is nice, i like this, adding it to my playbook

Reply:
Can't take credit! @mlclay came up with this approach when configuring our truffle project